### DISCUSSION TITLE
Allow User Message template to utilize layouts for cohesive styling, header & footer for system messages

### DISCUSSION THREAD

**Discussion by pyrobob83** [ExpressionEngine/ExpressionEngine#3582] [Feature Requests]
### Suggest an idea for ExpressionEngine

Currently the User Messages template at ~cp/design/system is very limited and cannot utilize layouts.  The desire is to allow system messages to be output inside a global site wrapper to maintain the header, footer and existing styling, but it doesn't work:

![image](https://github.com/ExpressionEngine/ExpressionEngine/assets/8193519/6a67ef56-2fdb-4710-a4ac-056fcdfebce2)

**Comment by intoeetive** [comment]
@pyrobob83 this feature already exists :)

https://docs.expressionengine.com/latest/control-panel/template-manager.html#custom-system-messages

> you simply need to create a `system_messages` template group, and create a template named `generic` within the group.

**Reply by pyrobob83** [reply]
Wow, totally missed that, it works great - thanks @intoeetive!  @robinsowell worth point out as we were talking about this in Slack.
